roles/openshift_prometheus/vars/default_images.yml

```diff
@@ -2,7 +2,7 @@
 # image prefix defaults
 l_openshift_prometheus_image_prefix: "{{ openshift_prometheus_image_prefix | default('openshift/') }}"
 l_openshift_prometheus_proxy_image_prefix: "{{ openshift_prometheus_proxy_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
-l_openshift_prometheus_alertmanager_image_prefix: "{{ openshift_prometheus_altermanager_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
+l_openshift_prometheus_alertmanager_image_prefix: "{{ openshift_prometheus_alertmanager_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
```

Comment:
Need it here too https://github.com/openshift/openshift-ansible/blob/9aaa59d7fb348311ba815296e79053faa3556860/roles/openshift_prometheus/vars/openshift-enterprise.yml#L5

Reply:
done!